packages/iris-grid/src/GotoRow.tsx

```diff
@@ -86,10 +87,10 @@ function GotoRow({
     ({ columns } = model.table);
   }
 
-  const res = 'Row number';
-
   const { rowCount } = model;
 
+  const gotoRowInputId = `goto-row-input-${shortid()}`;
```

Comment:
Right now this id is updating on every render. You should memoize this, something like:
```
  const gotoRowInputId = useMemo(() => `goto-row-input-${shortid()}`, []);
```

Otherwise the ID changes with every re-render/anytime a prop changes, which is unnecessary.